src/ExtInterface/ReefMod/Domain.jl

```diff
@@ -309,12 +309,26 @@ function load_initial_cover(::Type{ReefModDomain}, data_path::String, loc_ids::V
         icc_data[:, :, i] = Matrix(CSV.read(fn, DataFrame; drop=[1], header=false, comment="#"))
     end
 
-    # Take the mean over repeats, as suggested by YM (pers comm. 2023-02-27 12:40pm AEDT)
-    # Convert from percent to relative values
-    icc_data = dropdims(mean(icc_data, dims=2), dims=2) ./ 100.0
+    # Use ReefMod distribution for coral size class population (shape parameters have units log(cm^2))
+    # as suggested by YM (pers comm. 2023-08-08 12:55pm AEDT). Distribution is used to split ReefMod initial 
```

Comment:
That time stamp looks suspicious. If it was this month, it can't have been AEDT?

```suggestion
    # as suggested by YM (pers comm. 2023-08-08 12:55pm AEST). Distribution is used to split ReefMod initial 
```